### PR TITLE
[StickyListHeaders] to Nuget.

### PR DESCRIPTION
**Pull request by adlair**

### CODE REVIEW

File: Android/StickyListHeaders/component/component.yaml [hunk 1]
```diff
@@ -21,4 +21,15 @@ libraries:
...
   - name: "Android Sample"
     path: ../samples/StickyListHeadersSample.sln
- 
\ No newline at end of file
+    removeProjects:
+        - StickyListHeaders
+    installNuGets:
+      - project: StickyListHeadersSample
+        packages:
+          - Xamarin.StickyListHeaders
```
Comment (newky2k) [inline Android/StickyListHeaders/component/component.yaml]: The sample is not adding the library from the nuget package

File: Android/StickyListHeaders/nuget/Xamarin.StickyListHeaders.nuspec [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>StickyListHeaders is an Android library that makes it easy to integrate section headers in your ListView.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=871806</projectUrl>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=871805</licenseUrl>
+  </metadata>
+  <files>
+    <file src="output/StickyListHeaders.dll" target="lib/Xamarin.Android" />
```
Comment (newky2k) [inline Android/StickyListHeaders/nuget/Xamarin.StickyListHeaders.nuspec:16]: This is the wrong target name - to should be MonoAndroid60 or MonoAndroid80, what ever the minimum version of android is

**Review by newky2k — CHANGES_REQUESTED** [review]

**Comment by adlair** [comment]
I added new changes in the nest PR: https://github.com/xamarin/XamarinComponents/pull/362.

**Comment by bholmes** [comment]
There was a problem with Git and GitHub where the PR was disconnected from the fork.  This PR is patched and opened again in #362.

**Pull request closed without merging**